ek9000App/src/scripts/generate.py

```diff
@@ -133,10 +135,16 @@ def set_default_bi(self):
         self.vals['ONAM'] = 'high'
         self.vals['SCAN'] = 'I/O Intr'
 
+    def set_default_mbbi(self):
+        self.vals['SCAN'] = 'I/O Intr'
+        
+    def set_default_mbbo(self):
+        self.vals['PINI'] = 'YES'
+
     def set_default_bo(self):
         self.vals['ZNAM'] = 'low'
         self.vals['ONAM'] = 'high'
-        self.vals['PINI'] = 'YES'  # TODO: this seems wrong
+        self.vals['PINI'] = 'YES'
```

Comment:
Right, okay. I interpreted the comment as "YES seems like an invalid choice"

I'll take a look at the docs then

Reply:
I think your analysis is correct here. I suspect this hasn't been reported as an issue because most or all of our (admittedly few) ek9k deployments have the watchdog timer enabled with set-to-zero, so outputs get set to 0 regardless when restarting the IOC.